edb/server/compiler/compiler.py

```diff
@@ -2139,6 +2139,11 @@ async def describe_database_restore(
         )
         ctx.state.start_tx()
 
+        dump_with_extraneous_computables = (
+            dump_server_ver is not None
```

Comment:
Why version could be `None`? I would say it's better to assert on it, this condition sounds like we have "always latest version" dumps, which is probably nonsensical.

Reply:
I'm also curious why it can be `None` -- perhaps add a comment at least?

Reply:
The earliest versions did not have the version header in dumps. I'll add a comment

Reply:
Then `None` should be treated as a smaller version not larger. I.e. `dump_with_extraneous_computables` should be `True` for `None` not `False`.

Reply:
Ah, I see you fixed this.